nomad/structs/structs.go

```diff
@@ -9608,11 +9602,9 @@ func (a *AllocListStub) SetEventDisplayMessages() {
 }
 
 func setDisplayMsg(taskStates map[string]*TaskState) {
-	if taskStates != nil {
```

Comment:
Is this guaranteed here? (As opposed to being an empty map.)

Reply:
I don't think it matters whether it's nil or empty, does it? `range` should handle it either way.

Reply:
🤦 you're right. Nevermind.